src/main/java/hudson/cli/CLICommandInvoker.java

```diff
@@ -112,6 +120,8 @@ public Result invokeWithArgs(final String... args) {
 
     public Result invoke() {
 
+        Result result;
+        Error executionError = null;
```

Comment:
Seems that `executionError` is unused, no?

Also I see no reason why `result` is being defined here. Would seem simpler for end of method to read

```java
restoreAuth();
return new Result(returnCode, out, err);
```